commitlint.config.ts

```diff
@@ -558,7 +560,27 @@ module.exports = {
                         `Please use full URLs instead of #XYZ refs.`
                     ];
                 },
+    
+                'proper-revert-message': ({header, body}: {header: any, body: any}) => {
+                    let offence = false;
+
+                    // does msg have a body?
+                    if (header.match(/^[Rr]evert/) !== null) {
```

Comment:
this is wrong, when you do `git revert`, the commit message starts with uppercase, not lowercase